app/src/main/java/com/kelvin/bdaypost/birthday/data/ContactRepository.kt

```diff
@@ -0,0 +1,59 @@
+package com.kelvin.bdaypost.birthday.data
+
+import com.kelvin.bdaypost.birthday.data.model.Birthday
+import com.kelvin.bdaypost.birthday.data.model.Birthday.Companion.getDayOfYear
+import com.kelvin.bdaypost.data.Result
+import com.kelvin.bdaypost.birthday.data.model.ContactInfo
+import timber.log.Timber
+
+class ContactRepository private constructor(
```

Comment:
I think we could use Hilt to make this a singleton without needing the companion object, although I haven't tried it yet :P